balto-api/utils/tokens.js

```diff
@@ -0,0 +1,28 @@
+const jwt = require("jsonwebtoken")
+const { SECRET_KEY } = require("../config")
+
+const generateToken = (data) => jwt.sign(data, SECRET_KEY, { algorithm:"HS256", expiresIn: "24h" })
+
+const createUserJwt = (user) => {
+    const payload = {
+        email: user.email,
+        username: user.username,
```

Comment:
Hey Annesa, there isn't a username field in the schema — maybe we can talk about if we want to add one

Reply:
I think we're good without a username, thank you for catching that! I'll take it out.